### PR TITLE
Spanish UI string added, English string fix and Video Quality string added.

### PR DESCRIPTION
**Pull request by LetrixZ**

### CODE REVIEW

**Comment by LetrixZ** [comment]
Also, want to know if its possible to localize the other strings present in the app: the titles (Recent Sub, Movies, Categories) presented in utils/Utils.kt, and the provided by the web GET. Thanks.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2,12 +2,12 @@
     <string name="app_name">Anime X Stream</string>
     <string name="discover">Browse</string>
     <string name="retry">retry</string>
-    <string name="quality_auto">Auto</string>
+    <string name="quality_auto" translatable="false">Auto</string>
     <string name="released">Released:</string>
     <string name="type">Type:</string>
     <string name="status">Status:</string>
     <string name="next_episode"><u>Next Episode</u>></string>
-    <string name="previous_episode">&lt;<u>Previous Episode</u></string>
```
Comment (mukul500) [inline app/src/main/res/values/strings.xml:10]: This one &lt is used for '<' this arrow.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Anime X Stream</string>
+    <string name="discover">Descubre</string>
+    <string name="retry">reintentar</string>
+    <string name="released">Fecha de lanzamiento:</string>
+    <string name="type">Tipo:</string>
+    <string name="status">Estado:</string>
+    <string name="next_episode"><u>Episodio siguiente</u></string>
```
Comment (mukul500) [inline app/src/main/res/values-es/strings.xml:9]: On Next episode add '>' in end of string

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <string name="previous_episode"><u>Episodio anterior</u></string>
```
Comment (mukul500) [inline app/src/main/res/values-es/strings.xml:10]: In previous episode add '<' string

**Pull request merged**